### PR TITLE
Load dicom node from table

### PR DESCRIPTION
**Pull request by aauker**
data_processing/common/Container.py
* Move some infos to debugs
* add a “uid://<id>“ scheme to specify container auto-id instead of qualified address (It's useful to use auto-ids for testing, but it's hard to discriminate between integer ids (like slide IDs) and these auto-ids
* added `.address` attribute, keeping others for backwards capability

data_processing/common/custom_logger.py
* When you call `init_logger` with a new file handle, it gets added as an additional sink

data_processing/pipelines/*
* Bringing lung pipeline up to date with perturbations, 1.25mm pixels spacing 

data_processing/radiology/cli*
* Some line work compacting
* Move exception logger to `ERROR - {e}, stopping job execution...` for easier log parsing 

data_processing/radiology/cli/load_dicom.py
* Stop-gap cli tool to load dicoms from table

data_processing/radiology/mirp/imageProcess.py
* Target supervoxel size 100 mm^3
* When multiple ROIs are merged, use lowest label index as priority in the case of overlapping annotations post-process (Addition merge method created “fake” label regions when they overlapped, e.g. 3+4 was label 7, now is correctly label 3.)

data_processing/radiology/mirp/imageReaders.py
* Detect completely empty annotation/label files

data_processing/services/radiology_service.py
* Include a pipeline route
* Can see call to this route in cell 2 of {jupyper hub URI}/hub/user-redirect/lab/tree/lung-mind-project/lung-analysis-041321.ipynb

### CODE REVIEW

File: data_processing/common/custom_logger.py [hunk 1]
```diff
@@ -29,13 +31,16 @@ def init_logger(filename='data-processing.log'):
...
         fh.setFormatter(formatter)
         # add the handlers to logger
         logger.addHandler(ch)
         logger.addHandler(fh)
-
+    if logger.handlers:
+        fh = logging.FileHandler(log_file)
+        fh.setLevel(logging.INFO)
+        fh.setFormatter(formatter)
+        logger.addHandler(fh)
```
Comment (doori) [inline data_processing/common/custom_logger.py:43]: Looks like we have the same fh setup in the other if case. Should we move this outside the ifs?

File: data_processing/common/sparksession.py [hunk 2]
```diff
@@ -46,5 +46,6 @@ def spark_session(self, config_name, app_name):
 			.config("fs.defaultFS", "file:///") \
 			.config("spark.driver.extraJavaOptions", "-Dio.netty.tryReflectionSetAccessible=true") \
 			.config("spark.executor.extraJavaOptions", "-Dio.netty.tryReflectionSetAccessible=true") \
+			.config("spark.ui.enabled", "false") \
```
Comment (doori) [inline data_processing/common/sparksession.py:49]: curious why we want to disable the ui

**Comment by doori** [comment]
Great work! my understanding is that the scan containers are already in the graph database. Do we have some code to populate the scan containers from some data source? 
It would be nice to:
1) organize the jsons in pipelines under a subdirectory like "configs"
2) check exit code of the subprocesses in driver.py

File: data_processing/radiology/cli/load_dicom.py [hunk 3]
```diff
@@ -0,0 +1,71 @@
+'''
+Created: February 2021
+@author: aukermaa@mskcc.org
+
+Given a scan (container) ID
+1. resolve the path to the dicom folder
+2. for all dicoms, rescale into HU and optionally window
```
Comment (doori) [inline data_processing/radiology/cli/load_dicom.py:7]: is HU rescale still being done in this file, or has it been refactored?

File: data_processing/common/sparksession.py (hunk 2 above)
Comment (aauker) [reply data_processing/common/sparksession.py:49]: Oops, this snuck in, reverted!

File: data_processing/radiology/cli/load_dicom.py (hunk 3 above)
Comment (aauker) [reply data_processing/radiology/cli/load_dicom.py]: Ahh, fixed this doc string

**Pull request merged**